manifests/base/jupyter-datascience-notebook-imagestream.yaml

```
@@ -9,7 +9,7 @@ metadata:
     opendatahub.io/notebook-image-name: "Standard Data Science"
     opendatahub.io/notebook-image-desc: "Jupyter notebook image with a set of data science libraries that advanced AI/ML notebooks will use as a base image to provide a standard for libraries avialable in all notebooks"
     opendatahub.io/notebook-image-order: "20"
-  name: generic-data-science
+  name: s2i-generic-data-science-notebook
```

Probably we have to deprecate this `s2i`, since it is not that case any more, the same for the minimal one
```suggestion
  name: generic-data-science-notebook
```

---

we cant do that, as it created a different imagestream and would confuse the user.
check the image linked on the description.

---

Ahh, I see... that is fine then